mitodl/open-discussions
# Add UI for editing channel types

#### What are the relevant tickets?
Fixes #830 

#### What's this PR do?
Refactors the UI to look a little more similar to the mockup*. Updates backend to allow restricted type.

*See: https://projects.invisionapp.com/share/XSLE63T6GR2#/screens/303979450

#### How should this be manually tested?
As a moderator verify that you can change the channel type. You should also be able to edit the description in the appearance tab.

If you click Cancel then click the edit icon again the text should revert to what it was before.
